Packages/tests/UTF_HardwareHelperFunctions.ipf

```diff
@@ -1479,8 +1479,8 @@ Function AcquireData_NG(STRUCT DAQSettings &s, string device)
 			continue
 		endif
 
-		if(i >= NUM_ITC_TTL_BITS_PER_RACK                 \
-		   && (GetHardwareType(device) == HARDWARE_NI_DAC \
+		if(i >= NUM_ITC_TTL_BITS_PER_RACK                  \
+		   && (GetHardwareType(device) == HARDWARE_ITC_DAC \
 		       || HW_ITC_GetNumberOfRacks(device) < 2))
```

Comment:
`||` -> `&&`? We want to skip the TTL channels if we have:
- a ttl bit of rack one
- and itc hardware
- and only one rack

Reply:
good catch